tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -54,3 +74,20 @@ def test_get(self):
         self.assertIs(list_of_aspect.get('Body.Length'),
                       Metadata.CommitMessage.Body.Length)
         self.assertIsNone(list_of_aspect.get(Metadata))
+
+    def test_get_excludes(self):
+        self.assertIs(self.aspectlist_excludes.get(
+                      Metadata.CommitMessage.Shortlog.ColonExistence),
+                      Metadata.CommitMessage.Shortlog.ColonExistence)
+        self.assertIsNone(self.aspectlist_excludes.get(
+                          Metadata.CommitMessage.Shortlog.TrailingPeriod))
+        self.assertIsNone(self.aspectlist_excludes.get(
+                          Metadata.CommitMessage.Body.Existence))
+
+        self.assertEqual(self.instancelist_excludes.get(
+                      Metadata.CommitMessage.Shortlog.ColonExistence),
+                      Metadata.CommitMessage.Shortlog.ColonExistence('py'))
```

Comment:
Put this computed value in a variable before using assertEqual.
assert invocations shouldnt have complex calls in them.